logshipper/main.tf

```
@@ -0,0 +1,116 @@
+locals {
+  username = random_uuid.username.result
+  password = random_password.password.result
+  # The syslog_drain must be registered on the public domain for cloudfoundry.
+  # Cloud Foundry uses the syslog URL to route messages to the service.
+  # The syslog URL has a scheme of syslog, syslog-tls, or https, and can include a port number.
+  # More information here:
+  # https://docs.cloudfoundry.org/devguide/services/log-management.html
+  # https://docs.cloudfoundry.org/devguide/services/user-provided.html#syslog
+  syslog_drain = "https://${local.username}:${local.password}@${module.route.host}.app.cloud.gov/?drain-type=all"
```

[issue] if `var.domain` is passed into the module, then this url will be wrong.

```suggestion
  syslog_drain = "https://${local.username}:${local.password}@${module.route.endpoint}/?drain-type=all"
```